proposals/0005-MicroProfile-1.1.md

```diff
@@ -0,0 +1,61 @@
+# MicroProfile 1.1
+
+* Proposal: [MP-0005](0005-MicroProfile-1.1.md)
```

Comment:
As @johnament pointed out, MP-0005 is already taken. I would suggest MP-0008 as the number we're up to at the moment. I've re-titled other PRs to make this a bit more obvious.